### PR TITLE
feat(admin): build admin UI

### PR DESCRIPTION
**Pull request by ntu34543**

### CODE REVIEW

**Comment by ntu34543** [comment]
[[BE] [Admin] Build interface](https://trello.com/c/Cr1xVlGZ/59-be-admin-build-interface)

File: resources/views/admin/error.blade.php [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet"
+        href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+        crossorigin="anonymous">
+    <style>
+    @import url('https://fonts.googleapis.com/css?family=Montserrat:400,600,700');
+    @import url('https://fonts.googleapis.com/css?family=Catamaran:400,800');
+
+    .error-container {
```
Comment (uica-dev) [inline resources/views/admin/error.blade.php:19]: Should we create a new CSS file and put these styles into it? @ntu34543

File: routes/web.php [hunk 2]
```diff
@@ -13,6 +20,44 @@
 |
 */
 
+Route::group([
```
Comment (uica-dev) [inline routes/web.php:23]: Only admin can access to the Admin Portal, maybe related to Authorization @hoaingodong1

File: app/Models/Admin.php [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<?php
```
Comment (uica-dev) [inline app/Models/Admin.php:1]: Do we need this model and why?

**Review by uica-dev — COMMENTED** [review]
Left some comments, please address then I will approve! @ntu34543

File: resources/views/admin/error.blade.php (hunk 1 above)
Comment (ntu34543) [reply resources/views/admin/error.blade.php:19]: Done!

**Review by uica-dev — APPROVED** [review]

**Pull request merged**